Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.3-onbuild
+
+EXPOSE 9102
+EXPOSE 9125/udp
+
+VOLUME ["/mapping.conf"]
```

Comment:
I'm fine with removing it however, it doesn't force anyone to use a volume, just advertises that one can be placed there. Sorta a convenience item. The expose command is very similar, just advertises those ports they don't have to be used.

> On Apr 14, 2015, at 07:42, discordianfish <notifications@github.com> wrote:
> 
> In Dockerfile:
> 
> > @@ -0,0 +1,8 @@
> > +FROM golang:1.3-onbuild
> > +
> > +EXPOSE 9102
> > +EXPOSE 9125/udp
> > +
> > +VOLUME ["/mapping.conf"]
> I would omit this since it would force people to use a volume where I would rather create a custom image based on this one. You can still use a bind-mount even without this volume definition.
> 
> —
> Reply to this email directly or view it on GitHub.
>